packages/search-in-workspace/src/browser/search-in-workspace-frontend-contribution.ts

```diff
@@ -169,6 +179,22 @@ export class SearchInWorkspaceFrontendContribution extends AbstractViewContribut
             }
         });
 
+        commands.registerCommand(SearchInWorkspaceCommands.FOCUS_NEXT_RESULT, {
+            isEnabled: () => this.tryGetWidget()?.hasResultList() ?? false,
```

Comment:
We can simplify using:

```suggestion
            isEnabled: () => this.withWidget(undefined, widget => widget.hasResultList())
```

The same is true for `FOCUS_PREV_RESULT`.